src/lib/Behat/Component/TranslationDialog.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Behat\Component;
+
+use Behat\Mink\Session;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+
+class TranslationDialog extends Dialog
+{
+    private IbexaDropdown $ibexaDropdown;
+
+    public function __construct(Session $session, IbexaDropdown $ibexaDropdown)
+    {
+        parent::__construct($session);
+        $this->ibexaDropdown = $ibexaDropdown;
+    }
+
+    public function selectNewTranslation(string $languageName): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('expandNewTranslationDropdown'))->click();
+        $this->ibexaDropdown->verifyIsLoaded();
+        $this->ibexaDropdown->selectOption($languageName);
+    }
+
+    public function selectBaseTranslation(string $languageName): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('expandBaseTranslationDropdown'))->click();
+        $this->ibexaDropdown->verifyIsLoaded();
+        $this->ibexaDropdown->selectOption($languageName);
+    }
+
+    public function verifyIsLoaded(): void
+    {
+        $this->getHTMLPage()->setTimeout(3)
+            ->find($this->getLocator('addTranslationPopupModalTitle'))
+            ->assert()->textEquals('Create a new translation');
+    }
+
+    protected function specifyLocators(): array
+    {
+        return [
+            new VisibleCSSLocator('expandNewTranslationDropdown', '#add-translation-modal [for="add-translation_language"] + .ibexa-dropdown .ibexa-dropdown__selection-info'),
+            new VisibleCSSLocator('expandBaseTranslationDropdown', '#add-translation-modal [for="add-translation_base_language"] + .ibexa-dropdown .ibexa-dropdown__selection-info'),
+            new VisibleCSSLocator('dropdownItem', '.ibexa-dropdown__item .ibexa-dropdown__item-label'),
+            new VisibleCSSLocator('confirm', '.modal.show button[type="submit"],.modal.show button[data-click]'),
```

Comment:
`dropdownItem`, `confirm` and `decline` seem unused, I think we can remove them

Reply:
It seems that at least `confirm` is required: https://github.com/ibexa/admin-ui/actions/runs/3128355506/jobs/5076092792#step:13:122